src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/ChartAxisConfiguration.cs

```diff
@@ -8,25 +9,27 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Model.Chart
 {
     public class ChartAxisConfiguration
     {
-        public string Name;
+        public string Name = null!;
```

Comment:
Just wondering if we should also add the [Required] attribute to indicate that these fields are mandatory. That would hopefully cause a bad request error if the chart axis configuration is requested without a name, sortBy, title, unit etc.

Reply:
I am not sure which fields are or are not required here, so I've created another ticket for this rather than investigate this immediately.